notebooks/tel_and_site/subsys_req_ver/tma/SITCOMTN-067_Vel_Accel_Jerk_15Nov23.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #127.        if maxAzJerk &gt; 42.0 or maxElJerk &gt; 21.0 or maxAzAccel &gt; 10.5 or maxElAccel &gt; 5.25:
</pre><p>Possibly add in the velocity max limits. May be unnecessary since velocity is controlled but might be interesting if any do go over the limit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I removed the duplicate plots and just put in a print statement flagging the out of spec slews.  This had been in there for debug purposes.